ui_panels.py

```diff
@@ -112,11 +112,21 @@ def draw_upload_common(layout, props, asset_type, context):
         # row = layout.row()
         # row.enabled = False
         # row.prop(props, 'id', icon='FILE_TICK')
-    layout.prop(props, 'category')
-    if props.category != 'NONE' and props.subcategory != 'NONE':
-        layout.prop(props, 'subcategory')
-    if props.subcategory != 'NONE' and props.subcategory1 != 'NONE':
-        layout.prop(props, 'subcategory1')
+    row = layout.row()
+    if props.category == 'NONE':
+        row.alert =True
+    row.prop(props, 'category')
+    if props.category != 'NONE' and props.subcategory != 'EMPTY':
+        row = layout.row()
+        if props.subcategory =='NONE':
+            row.alert = True
+        row.prop(props, 'subcategory')
+
+    if props.subcategory != 'NONE' and props.subcategory1 != 'EMPTY':
```

Comment:
If in category and subcategory which has subcategory1, I fill some value for all three (e.g. Industrial-Tools-Handtool) and then I change my mind and choose a subcategory which does not have a subcategory1, the subcategory row does not dissapear and it prints out errors into console:

```
WARN (bpy.rna): source/blender/python/intern/bpy_rna.c:1333 pyrna_enum_to_py: current value '2' matches no enum in 'BlenderKitModelUploadProps', 'Dasdad', 'subcategory1'
```

Subcategory1 row is there without any string. I need to choose subcategory1 and in it choose option `Empty`. The row disapears and errors stop.